coops-ui/src/components/AnnualReport/Directors.vue

```diff
@@ -981,6 +1019,11 @@ export default class Directors extends Mixins(DateMixin, ExternalMixin, EntityFi
     // close standalone cessation date picker and reset date
     this.cessationDateTemp = null
     this.activeIndexCustomCease = null
+    console.log(this.directors)
+    if (this.directors.filter(x => x.cessationDate == null).length < 2) {
```

Comment:
I don't understand what this code is doing... looking for directors that aren't ceased?

Reply:
well, it shouldn't even be here, so my mistake...looking into it...possibly missed a "push"